integration/test/multireleasehandling/multi_release_handling_test.go

```diff
@@ -70,13 +70,32 @@ var releaseCR2 = &releasev1alpha1.Release{
 	},
 }
 
-// TestMultiReleaseHandling runs following steps:
+var releaseCycleCR1 = &releasev1alpha1.ReleaseCycle{
+	TypeMeta: releasev1alpha1.NewReleaseCycleTypeMeta(),
+	ObjectMeta: metav1.ObjectMeta{
+		Name: "aws.v6.1.0",
+	},
+	Spec: releasev1alpha1.ReleaseCycleSpec{
+		DisabledDate: releasev1alpha1.DeepCopyDate{Time: time.Date(2019, 4, 8, 0, 0, 0, 0, time.UTC)},
+		EnabledDate:  releasev1alpha1.DeepCopyDate{Time: time.Date(2019, 1, 8, 0, 0, 0, 0, time.UTC)},
+		Phase:        releasev1alpha1.CyclePhaseEOL,
+	},
+}
+
+// TestMultiReleaseHandling makes sure that App CRs for components shared
+// between EOL and non-EOL Release CRs are not removed.
```

Comment:
@xh3b4sd let me know if this is what you're looking for.

Reply:
> TestMultiReleaseHandling makes sure that App CRs for components shared between different Releases are not removed when one release transitions to its final EOL phase.

What about this? The EOL/non-EOL lingo is something you need to know. When I look at this in half a year it becomes super hard to recall all the details.

Reply:
Sure. I don't see how one can not be into EOL lingo while trying understand this code though.